lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -1054,10 +1069,34 @@ def control_state(vm, vms_service, module):
             )
 
 
+def wait_for_import(module, vms_module, vms_service):
+    if module.params['wait']:
+        start = time.time()
+        timeout = module.params['timeout']
+        poll_interval = module.params['poll_interval']
+        while time.time() < start + timeout:
+            vm = _fetch_vm(module, vms_module, vms_service)
+            if vm:
+                return vm
+            time.sleep(poll_interval)
+
+
+def _fetch_vm(module, vms_module, vms_service):
+    if (module.params['id'] is not None):
+        # The vm id can not be searched with vms_module.search_entity
+        # since the 'id' attribute is not searchable in the engine, therefore searching
+        # the id through vms_service.vm_service.
+        vm_service = vms_service.vm_service(module.params['id'])
+        vm = vm_service.get()
+    elif (module.params['name'] is not None):
+        vm = vms_module.search_entity()
+    return vm
+
+
 def main():
     argument_spec = ovirt_full_argument_spec(
         state=dict(
-            choices=['running', 'stopped', 'present', 'absent', 'suspended', 'next_run'],
+            choices=['running', 'stopped', 'present', 'absent', 'suspended', 'next_run', 'registered'],
```

Comment:
Can you please add a note in documentation that `registered` is supported since 2.4?